src/pages/AppDetail.svelte

```diff
@@ -35,7 +26,9 @@
     <Pill message="Deprecated" bgColor="#4a5568" />
   {/if}
   <p>{app.description}</p>
-  <table class="mzp-u-data-table">
+  <table>
```

Comment:
I opted not to use Protocol's table here because its default styling was not suitable for this particular table, and it was more work than needed to override it.

Reply:
Makes sense! We should use protocol where it's helpful. I agree your styling makes more sense in this context.